src/main/java/com/dougnoel/sentinel/elements/tables/Table.java

```diff
@@ -973,4 +974,23 @@ public boolean waitForSpecificCellToContain(int numberOfSecondsToWait, String co
 			return false;
 		}
 	}
+
+	/**
+	 * Compares all values in the given column to the given referenceNumber, using the given comparisonType.
+	 * Converts all values in the given column to double.
+	 * Valid comparisonType values = {"less than", "greater than"}. Any other value of comparisonType will make this method perform an "equals" comparison.
+	 * @param columnHeader String name of the column
+	 * @param comparisonType String type of comparison to perform. Valid values are "less than", "greater than". Any other string will force an "equals" comparison.
+	 * @param referenceNumber double the number to compare against.
+	 * @return boolean true if all values in the column satisfy the given comparison. false otherwise.
+	 */
+	public boolean verifyNumericValuesInWholeColumn(String columnHeader, String comparisonType, double referenceNumber){
+		var allColumnData = getAllCellDataForColumn(columnHeader).stream().map(Double::parseDouble).collect(Collectors.toList());
+		if(comparisonType.equalsIgnoreCase("less than"))
```

Comment:
Do this as a case statement. You can filter the comparisonType through to make it all lowercase before the comparison. This will be easier to read and easier for someone else to modify later if they want to add stuff like greater than or equal to or not equal to.